scripts/setup-flex-builddir

```diff
@@ -275,7 +275,7 @@ setup_builddir () {
         existing_builddir=0
     fi
 
-    mkdir -p $BUILDDIR/conf
+    mkdir -p $BUILDDIR/conf/multiconfig
```

Comment:
I think we don't need multiconfig in the build folder. If we have it in layer as well it will work. Can you check that. If it is true then we don't need this change and other changes in this file

Reply:
Yes, that's true. we can have multiconfig in our layer as well. But we are separating out build configurable parameters for dom0 and domu in their respective conf files instead of having them all mixed up in local.conf file. That's why we put these multiconfig files in build folder. I think having user to change build parameters from layer dir does not looks feasible. Thoughts?